modules/nf-core/basicpy/meta.yml

```diff
@@ -0,0 +1,46 @@
+name: "basicpy"
+description: write your description here
+keywords: illumiation_correction, microscopy, imaging
+  - sort
+tools:
+  - "basicpy":
+      description: "BaSiCPy is a python package for background and shading correction of optical microscopy images. It is developed based on the Matlab version of BaSiC tool with major improvements in the algorithm."
+      homepage: "https://github.com/peng-lab/BaSiCPy"
+      documentation: "https://basicpy.readthedocs.io/en/latest/index.html"
+      tool_dev_url: "https://github.com/peng-lab/BaSiCPy"
+      doi: "doi: 10.1038/ncomms14836."
+      licence: "MIT License"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+
+  - image:
+      type: file
+      description: Tiff file to be used for dark and flat field illumination correction
+      pattern: "*.{tiff,tif}"
+
```

Comment:
`cpu_gpu` is missing :)

Reply:
I added it here. What is the proper way to have a mandatory input parameter though? Documentation for modules says no values as MANDATORY input channels...